app2/src/lib/wallet/utilities/format.ts

```diff
@@ -0,0 +1,8 @@
+export const truncateUnionAddress = (address: string, length = 6) =>
+  length > 0 ? `${address.slice(0, length)}...${address.slice(-length)}` : address
+
+export const truncateEvmAddress = (address: string, length = 6) =>
+  length > 0 ? `${address.slice(0, length)}...${address.slice(-length)}` : address
+
+export const convertByteArrayToHex = (byteArray: Uint8Array): string =>
+  byteArray.reduce((hex, byte) => hex + byte.toString(16).padStart(2, "0"), "").toUpperCase()
```

Comment:
let's delete these for now and display untruncated values, and introduce a proper truncate svelte component